### ISSUE TITLE
Training video still referencing closed comment

### ISSUE THREAD

**Issue by jameshenrylittle** [github/training-kit#479]
## Overview
**TL;DR**
The youtube video at https://www.youtube.com/watch?v=kocHKaHtZCc is still asking briannamaries comment to be opened in the tutorial but the comment is closed and the label for it has been deleted.

#### Screenshots
<If bug fix, please upload images of the error to better showcase the problem.>

#### Reasoning
<If new idea or direction, please create a checklist of proposed ideas or next steps. [Example](https://github.com/github/training-kit/issues/415).>

### How to Contribute
- [ ] Comment on this issue if you'd like to work/collaborate on it! 
- [ ] Fork the [repository](https://github.com/github/training-kit).
- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://services.github.com/on-demand/intro-to-github/) is a refresher course if you're unsure about how to make a change on GitHub.
- [ ] Push your changes to your repository. 
- [ ] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/on-demand`.

### Questions?
- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!

**Comment by hollenberry** [comment]
@jameshenrylittle Thank you for the notice! I'll update our video to reflect current directions.
